js/src/forum/components/SuspensionInfoModal.js

```diff
@@ -40,4 +40,9 @@ export default class SuspensionInfoModal extends Modal {
       </div>
     );
   }
+
+  hide() {
+    localStorage.setItem(localStorageKey(), this.attrs.until);
+    this.attrs.state.close();
+  }
```

Comment:
for some reason this isn't working for me, the local storage is correctly being set, but the modal still shows on every page load, the value of the local storage item is always the same

`Fri Jan 01 2038 22:03:46 GMT+0100 (Central European Standard Time)`

Reply:
Hmm, checking...

Reply:
Might need to use `.getTime()` and compare the UTC timestamps